code/modules/mob/living/carbon/human/human.dm

```diff
@@ -190,7 +195,7 @@
 	if(stat == DEAD)	return
 	show_message("<span class='userdanger'>The blob attacks you!</span>")
 	var/dam_zone = pick("chest", "l_hand", "r_hand", "l_leg", "r_leg")
-	var/obj/item/organ/limb/affecting = get_organ(ran_zone(dam_zone))
+	var/obj/item/organ/limb/affecting = getrandomorgan(dam_zone)
```

Comment:
umm you sure this line is correct?i though getrandomorgan needed an human,not a dam zone